.github/workflows/tests.yml

```diff
@@ -54,7 +65,7 @@ jobs:
     if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     container:
-      image: faasm/cpp-sysroot:0.2.0
+      image: faasm/cpp-sysroot:0.2.1
```

Comment:
The motivation for this PR is that I realised that these versions were out of sync, and I have been forgetting often lately :facepalm: